presto-main/src/main/java/com/facebook/presto/util/GcStatusMonitor.java

```diff
@@ -165,7 +165,6 @@ private void logTaskInfos(List<SqlTask> sqlTasks)
         ArrayList<List<String>> taskLogInfoList = new ArrayList<>();
         taskLogInfoList.add(Arrays.asList(
                 "Task ID",
-                "Query ID",
```

Comment:
I think you will end up needing this if we list all the queries first without interleaving them with their tasks. Sorry! I am learning more about what we need as I review this.